apis/infix/atrium-api-infix/src/commonMain/kotlin/ch/tutteli/atrium/api/infix/en_GB/listExpectations.kt

```diff
@@ -0,0 +1,21 @@
+package ch.tutteli.atrium.api.infix.en_GB
+
+import ch.tutteli.atrium.api.infix.en_GB.creating.IndexWithCreator
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.logic._logic
+import ch.tutteli.atrium.logic.last
+
+/**
+ * Expects that the subject of `this` expectation (a [List]) is not empty and that the last element
+ * in the list holds all assertions the given [IndexWithCreator.assertionCreator] creates for it.
```

Comment:
```suggestion
 * in the list holds all assertions the given [assertionCreator] creates for it.
```
We also don't expect IndexWithCreator here